charts/konk/templates/init-pod.yaml

```diff
@@ -29,3 +29,4 @@ spec:
       name: {{ include "konk.fullname" . }}-scripts
       defaultMode: 0777
   restartPolicy: OnFailure
+  terminationGracePeriodSeconds: 10
```

Comment:
Do we need to add this?

Reply:
Need? Not really, but it speeds up termination of the pod. The default is 30 seconds, and the script doesn't react to SIGTERM, so we always have to wait for the termination grace period for the pod to terminate.